src/ops/embedding.cc

```diff
@@ -101,8 +101,8 @@ bool EmbeddingParams::is_valid(ParallelTensorShape const &input) const {
 
```

Comment:
Is there any way you can make this diff more readable? Is there some merge conflict that needs to be addressed?

Reply:
That is strange, I don't see any conflicts on my end. I will take a detailed look and see what the issue is.